src/rt/sections_android.d

```diff
@@ -63,11 +63,11 @@ void initSections() nothrow @nogc
 {
     pthread_key_create(&_tlsKey, null);
 
-    auto mbeg = cast(immutable ModuleInfo**)&__start_minfo;
-    auto mend = cast(immutable ModuleInfo**)&__stop_minfo;
+    auto mbeg = cast(immutable ModuleInfo**)&__start___minfo;
+    auto mend = cast(immutable ModuleInfo**)&__stop___minfo;
```

Comment:
I don't know where these magic linker symbols come from, a `grep` of the Android NDK for them turns up nothing.  But these are the symbols ldc uses and the extra underscores are needed for this to actually work with the NDK's linker.